libpod/oci_conmon_linux.go

```diff
@@ -1016,6 +1017,12 @@ func (r *ConmonOCIRuntime) createOCIContainer(ctr *Container, restoreOptions *Co
 		// conmon not having a pid file is a valid state, so don't set it if we don't have it
 		logrus.Infof("Got Conmon PID as %d", conmonPID)
 		ctr.state.ConmonPID = conmonPID
+		sent, err2 := daemon.SdNotify(false, fmt.Sprintf("MAINPID=%d\n", conmonPID))
+		if err2 != nil {
```

Comment:
sent doesn't seem to be used?

You could write as:
```suggestion
		if _, err := daemon.SdNotify(false, fmt.Sprintf("MAINPID=%d\n", conmonPID)); err != nil {
```